### ISSUE TITLE
Cannot validate with basepath option if file name contains square brackets

### ISSUE THREAD

**Issue by rishubil** [frictionlessdata/frictionless-py#1206]
Cannot validate with basepath option if file name contains square brackets.

I think `is_expandable_path` is the cause.

https://github.com/frictionlessdata/frictionless-py/blob/6c120a1716abc8c5a825a88c62a91ba0935be4b1/frictionless/helpers.py#L228-L234

`is_expandable_path` checks `glob.has_magic`, which seems to recognize the brackets contained in the actual filename as the pattern used by `glob`.

The following shows the files and commands used for testing.

```console
$ tree
.
├── resource1.json
├── resource2.json
└── test
    ├── [file].csv
    └── file.csv

1 directory, 4 files

$ cat resource1.json 
{
  "path": "file.csv",
  "name": "test",
  "format": "csv",
  "schema": {
    "fields": [
      {
        "type": "integer",
        "name": "id"
      }
    ]
  }
}

$ cat resource2.json
{
  "path": "[file].csv",
  "name": "test",
  "format": "csv",
  "schema": {
    "fields": [
      {
        "type": "integer",
        "name": "id"
      }
    ]
  }
}

$ cat test/file.csv 
id
1

$ cat test/\[file\].csv 
id
1

$ frictionless validate resource1.json --type resource --basepath ./test

# -----
# valid: file.csv 
# -----

## Summary 

+------------------------+-------------------+
| Description            | Size/Name/Count   |
+========================+===================+
| File name              | file.csv          |
+------------------------+-------------------+
| File size (bytes)      | 4                 |
+------------------------+-------------------+
| Total Time Taken (sec) | 0.046             |
+------------------------+-------------------+


$ frictionless validate resource2.json --type resource --basepath ./test

# -------
# invalid: [file].csv 
# -------

## Summary 

+-----------------------------+-------------------+
| Description                 | Size/Name/Count   |
+=============================+===================+
| File name (Not Found)       | [file].csv        |
+-----------------------------+-------------------+
| File size                   | N/A               |
+-----------------------------+-------------------+
| Total Time Taken (sec)      | 0.044             |
+-----------------------------+-------------------+
| Total Errors                | 1                 |
+-----------------------------+-------------------+
| Scheme Error (scheme-error) | 1                 |
+-----------------------------+-------------------+

## Errors 

+-------+---------+---------+---------------------------------------------------+
| row   | field   | code    | message                                           |
+=======+=========+=========+===================================================+
|       |         | scheme- | The data source could not be successfully loaded: |
|       |         | error   | [Errno 2] No such file or directory: '[file].csv' |
+-------+---------+---------+---------------------------------------------------+
```

**Comment by roll** [comment]
Thanks @rishubil we will fix it

**Comment by roll** [comment]
It's been fixed in v5 branch in #1119 (we're about to release this version)
